react/slider/src/style.css

```diff
@@ -0,0 +1,163 @@
+.w-slider__wrapper {
+  display: grid;
+  max-width: 100%;
+  margin: 8px 0px;
+}
+.w-slider__input-wrapper {
+  grid-row: 1;
+  grid-column: 1;
+  display: grid;
+}
+input[type="range"] {
```

Comment:
All selectors should prob be prefixed with .w-slider or whatever is a unique thing for only this component. if not, these lines will hit all other input type range anywhere else on the same page.  Probably also better to just give these a class instead of using the attribute selectors (something like .w-slider__input-range is better, and would also work for the weird pseudo stuff like .w-slider__input-range::-webkit-slider-thumb ).